onyo/lib/consts.py

```diff
@@ -46,3 +47,32 @@
 
 Use for type matching (:py:func:`onyo_get`) and output.
 """
+
+ONYO_DIR = Path('.onyo')
+r"""Path of the onyo directory.
+
+This the dir within a repo that contains the onyo
+configuration, templates, etc.
+"""
+
+ONYO_CONFIG = ONYO_DIR / 'config'
+r"""Path of the Onyo config file.
+"""
```

Comment:
Nit: `ruff` doesn't like these training lines. Please put on one line for docstrings with only a summary.

I know these were in the originals. That's my bad.